package/src/contexts/themeContext/utils/theme.ts

```diff
@@ -748,6 +752,10 @@ export const defaultTheme: Theme = {
     },
     typingIndicatorContainer: {},
   },
+  MessagePinnedHeader: {
```

Comment:
This needs to be nested inside `messageSimple` property. Since in heirarchy, its a child of MessageSimple component. So something like:

```tsx
 messageSimple: {
     pinnedHeader: {
       container: {},
       label: {}
     },
      ...
}
```